en/index.html

```diff
@@ -82,7 +82,7 @@
       <div class="mx-auto">
         <div class="flex justify-between items-center ">
           <div class="flex w-full flex-wrap md:flex-nowrap border-b border-black">
-            <a href="#" class="mx-4 items-center basis-1/5 flex">
+            <a href="../en" class="mx-4 items-center basis-1/5 flex">
```

Comment:
Why only this link is different?

Reply:
Sorry sir. I updated link and remove link in logo for Tokutei.